saphanabootstrap-formula.spec

```diff
@@ -44,13 +55,40 @@ SAP HANA deployment salt formula
 pwd
 mkdir -p %{buildroot}/srv/salt/
 cp -R %{fname} %{buildroot}/srv/salt/%{fname}
+cp -R %{ftemplates} %{buildroot}/srv/salt/%{fname}/%{ftemplates}
+
+# SUMA Specific
+mkdir -p %{buildroot}/usr/share/susemanager/formulas/states/%{fname}
```

Comment:
@arbulu89 I suggest to change the /usr/share to the macro %{_datadir}

Reply:
Ok! I will copy that from habootstrap-formula